x-pack/plugins/actions/server/plugin.ts

```diff
@@ -151,8 +154,14 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
 
     // get executions count
     const taskRunnerFactory = new TaskRunnerFactory(actionExecutor);
-    const actionsConfigUtils = getActionsConfigurationUtilities(
-      (await this.config) as ActionsConfig
+    const actionsConfig = (await this.config) as ActionsConfig;
+    const actionsConfigUtils = getActionsConfigurationUtilities(actionsConfig);
+
+    this.preconfiguredActions.push(
```

Comment:
I think we should probably be running these through the actionType validators, to make sure the config/secrets get validated by the actionType.  I don't think that's happening right now.  I made some comments in my earlier review about other validations we should be doing, I think it's fine to do them all in a separate issue/PR.